docs/installation/webserver/nginx.md

```diff
@@ -6,106 +6,263 @@ Nginx: Configuring the virtual host
+
+    try_files                  $uri $uri/ /index.php?$query_string;
+}
+
+# Don't log, and do cache, asset files
+location ~* ^.+\.(?:atom|bmp|bz2|css|doc|eot|exe|gif|gz|ico|jpe?g|jpeg|jpg|js|map|mid|midi|mp4|ogg|ogv|otf|png|ppt|rar|rtf|svg|svgz|tar|tgz|ttf|wav|woff|woff2|xls|zip)$ {
+    access_log                 off;
+    log_not_found              off;
+    expires                    max;
+    add_header                 Pragma public;
+    add_header                 Cache-Control "public, mustrevalidate, proxy-revalidate";
+}
+
+# Don't create logs for favicon.ico, robots.txt, or rss.jpg requests
+location = /(?:favicon.ico|robots.txt|rss.jpg) {
+    log_not_found              off;
+    access_log                 off;
+}
 ```
-fastcgi_pass unix:/path/to/php-fpm/socket.sock;
+
+
+#### `restrictions.conf`
+
+The `restrictions.conf` file defines a common set of restrictions for a Bolt
+site.
+
+```nginx
+# Block access to "hidden" files, i.e. file names that begin with a dot
+location ~ /\. {
+    deny                      all;
+}
+
+# Block PHP files from being run in upload (files), app, theme and extension directories
+location ~* /(?:extensions|files|theme)/(.*)\.php$ {
+    deny                      all;
+}
+
+# Block access to Sqlite database files, Apache .htaccess & .htpasswd files
+location ~ /\.(?:db|htaccess|htpasswd)$ {
+    deny                      all;
+}
+
+# Block access to the app, cache & vendor directories
+location ~ /(?:app|src|tests|vendor)/ {
+    deny                      all;
+}
+
+# Block access to Markdown, Twig & YAML files directly
+location ~* /(.*)\.(?:markdown|md|twig|yaml|yml)$ {
+    deny                      all;
+}
+
+# Block access to the app, cache & vendor directories
+#
+# NOTE: If you have one or more of 'app' , 'src', 'test' and 'vendor' as
+# sub-directories of your installation root, you should uncomment
+# this location block to prevent site visitors having access to the
+# various directories that contain executable code.
+#
+# NOTE: This approach is *not* recommended for production use.
+#
+#location ~ /(?:app|cache|src|tests|vendor)/(.*)$ {
+#    deny all;
+#}
+```
+
+
+#### `php-fpm.conf`
+
+The `php-fpm.conf` file define the settings for the PHP FastCGI Process Manager
+used for you Bolt site(s).
+
+```nginx
+location ~ \.php$ {
+    include                    fastcgi_params;
+    fastcgi_param              SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    fastcgi_param              HTTP_PROXY "";
+    fastcgi_index              index.php;
+
+    # If using UNIX sockets UPDATE and uncomment the next line
+    #fastcgi_pass               unix:/run/php-fpm/www.sock;
+
+    # If using TCP sockets uncomment the next line
+    #fastcgi_pass               127.0.0.1:9000;
+}
+```
+
+
+On larger multi-CPU hosts with several busy sites, you will no doubt want to
+use several FPM pools, with each pool defining their own socket. You can simply
```

Comment:
Why? the worker threads are spread over the cores/CPU's anyway just like nginx worker threads are, right?

Isn't the advantage of multiple pools that you can run them as different users and therefore get security isolation?